rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java

```diff
@@ -440,6 +438,26 @@ void addInputFromOutputSentToScript_withMatchingOutputScript_shouldAddInputWithO
         Assertions.assertEquals(newTransactionInput.getOutpoint().getHash(), sourceTransactionOutput.getParentTransactionHash());
     }
 
+    @Test
+    void addP2SHEmptyInputScript_shouldAddExpectedScriptSigToInput() {
+        // arrange
+        Federation federation = P2shErpFederationBuilder.builder().build();
+        Script redeemScript = federation.getRedeemScript();
+
+        BtcTransaction transaction = new BtcTransaction(btcMainnetParams);
+        transaction.addInput(transaction.getHash(), 0, redeemScript);
```

Comment:
This line raises a few questions
- Is that method expecting a redeem script or a script sig? I'd debug this and check the transaction after adding the input
- If it's expecting a redeemScript, does it use it to build an empty scriptsig? Does this work with ERP type scripts? i.e. adding the flo op code
- Is this how we are creating the spend tx? If that is the case then no need to add the empty script sig, right?